_layouts/default.html

```diff
@@ -5,6 +5,10 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="{{ '/assets/css/style.css?v=' | append: site.github.build_revision | relative_url }}">
+    <!-- Set theme-color. -->
+    <meta name="theme-color" content="#151515" />
```

Comment:
As I comment https://github.com/pages-themes/hacker/issues/67#issuecomment-870893961 on feature #67 

Could you apply the suggestion?

```
<meta name="theme-color" content="#151515">
<meta name="msapplication-navbutton-color" content="#151515">
<meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
```